vllm/core/policy.py

```diff
@@ -15,13 +15,14 @@ def get_priority(
     def sort_by_priority(
         self,
         now: float,
-        seq_groups: List[SequenceGroup],
-    ) -> List[SequenceGroup]:
-        return sorted(
-            seq_groups,
-            key=lambda seq_group: self.get_priority(now, seq_group),
-            reverse=True,
-        )
+        seq_groups: deque[SequenceGroup],
+    ) -> deque[SequenceGroup]:
```

Comment:
nit: Could you use `Deque` in `typing` package for type annotation? Same for other places.
```python
from typing import Deque
```